pyanaconda/ui/gui/spokes/storage.py

```diff
@@ -974,10 +974,10 @@ def _check_space_and_get_dialog(self, disks):
         log.debug("disk free: %s  fs free: %s  sw needs: %s  auto swap: %s",
                   disk_free, fs_free, required_space, auto_swap)
 
-        # compare only to 90% of disk space because fs takes some space for a metadata
-        if (disk_free * 0.9) >= required_space + auto_swap:
+        # We need enough space for the software, the swap and 10% of the disk space for metadata.
+        if disk_free >= required_space + auto_swap + 0.1 * disk_free:
```

Comment:
OK we are taking swap into account but what about `/boot`, `/home` and similar?

Reply:
Added a comment.